README.md

```diff
@@ -24,13 +24,20 @@ Wing programs can be executed locally (*yes, no internet required*) using a full
 
 Our mission is to bring back your creative flow and close the gap between imagination and creation. Wing elevates your cloud development experience to new heights (the puns are just inevitable so bear with us)! 🚀
 
+We fullfil this mission by reducing cognitive load and increasing iteration speeds. How? by enabling you to work at a higher level of abstraction, letting you focus on business logic instead of cloud mechanics, and write much less code (90%-95% less!). We also provide you with a set of tools that let you test your code 100 times faster than before.
```

Comment:
From Chris: remove the numbers until we can better defend them

Reply:
Alternative version:
We fulfill this mission by reducing cognitive load and increasing iteration speeds. How? by enabling you to work at a higher level of abstraction, letting you focus on business logic instead of cloud mechanics, and write much less code. We also provide you with a set of tools that let you test your code significantly faster than before.